src/ui/scss/component/_settings.scss

```diff
@@ -0,0 +1,9 @@
+.select-hours {
```

Comment:
This shouldn't need new styles. Please try to look at existing form elements to see how we can do this without adding new css

Reply:
I will look